### ISSUE TITLE
Stopped recording reports

### ISSUE THREAD

**Issue by glocalglocal** [mjaakko/NeoStumbler#355]
### Description

1.4.0 from F-Droid worked OK for days, but recently it stopped recording reports. I manually start scanning, the GPS pin appears next to the clock as usual, but nothing is recorded on the Reports screen.  

The app may have stopped recording when it upgraded to 1.4.1. A waited a few days, then downgraded to 1.4.0, and then it upgraded to 1.4.2 but it still fails to pick up any information. 

As far as I know, nothing else has changed in permissions etc. and I am not getting an error. 

### App variant

fdroid

### Version

1.4.2

### Device

Moto g73

### Android version

14

### New bug

- [X] I have checked that this bug has not been reported before

**Comment by mjaakko** [comment]
Hmm, interesting. There hasn't been any changes to how the scanner service works after 1.4.0.

It sounds like NeoStumbler can't get accurate locations for some reason and thus can't create reports. If you enable notifications for NeoStumbler and choose "Detailed" notification style from the settings, does the notification show the number of satellites in use? If you use [GPSTest](https://f-droid.org/packages/com.android.gpstest.osmdroid/), can you get an accurate location?

**Comment by glocalglocal** [comment]
Detailed notifications say '0/22 satellites in use'. GPSTest reports it sees 11/36 satellites and gets accurate location like several map apps I have installed.

**Comment by mjaakko** [comment]
Thanks! Seems that there's something weird going on. I'm not sure what would cause this and I can't reproduce this easily

Please keep me updated if you find anything else  about this and I'll try to investigate further

**Comment by glocalglocal** [comment]
I just tried again. The notification now says '21/33 satellites in use' but still no reports.

**Comment by mjaakko** [comment]
Are you using the motion sensor for motion detection?

**Comment by glocalglocal** [comment]
Yes, but I think 2 updates ago the problem disappeared. It's working again as expected. Mystery!

**Comment by mjaakko** [comment]
Great, thanks for the confirmation! The motion sensor tends to be quite unreliable

**Comment by glocalglocal** [comment]
Unfortunately, 2-3 versions and 7000 reports later the problem retuned with the current version. I tried all motion detections and my default is by location.

**Comment by glocalglocal** [comment]
> Unfortunately, 2-3 versions and 7000 reports later the problem retuned with the current version. I tried all motion detections and my default is by location.

And now v1.5.3 is working again. This may help figure out what is causing the problem.
